server/mailet/integration-testing/src/test/java/org/apache/james/smtp/SmtpRandomStoringTest.java

```diff
@@ -117,38 +106,38 @@ private void createUsersAndMailboxes() throws Exception {
         DataProbe dataProbe = jamesServer.getProbe(DataProbeImpl.class);
         dataProbe.addDomain(DEFAULT_DOMAIN);
         dataProbe.addUser(FROM, PASSWORD);
-        USERS.forEach(Throwing.<String>consumer(user -> {
+        USERS.forEach(user -> populateUser(mailboxes, dataProbe, user));
+    }
+
+    private void populateUser(MailboxProbeImpl mailboxes, DataProbe dataProbe, String user) {
```

Comment:
`s/MailboxProbeImpl mailboxes/MailboxProbeImpl mailboxProbe`

Also, that probes makes the signature ugly. Can't it be a field?

Reply:
Not for one initialization method